hackathon_site/dashboard/frontend/src/components/dashboard/TeamCard/TeamCard.js

```diff
@@ -0,0 +1,34 @@
+import React from "react";
```

Comment:
Also feel free to import your component into Dashboard.js and add it above the .map of DashCard

Reply:
would I have to add the props for Dashboard to do that? Since my component takes in members and team_id and a handlebutton function

Reply:
If you want to put in the effort to put in dummy values then go ahead

Reply:
uhhmm nah lol